lisa_commands/request_helper.py

```diff
@@ -127,6 +127,29 @@ def youtube_search(query):
         return f"https://www.youtube.com/watch?v={video_id}"
     else:
         return None
+
+def wikipedia_search(query):
+    """
+    Submit a search to wikipedia
+    :query what to query for
+    """
+    request_params = {
+    'action':'query',
+    'list':'search',
+    'srsearch': query,
+    'format':'json'
+    }
+
+    request = urllib.request.Request('https://en.wikipedia.org/w/api.php', request_params)
```

Comment:
2 things.
1. there should be a question mark at the end of the `api.php` part. That's required for the url to accept params. Check the youtube function url for an example of what I'm talking about.

2. it should be a URL + request _params (if you use a comma request assumes that it's a POST request and the params are the body which is something we don't want.

So it should look something like this:
request = urllib.request.Request('https://en.wikipedia.org/w/api.php?' + request_params )